modular_skyrat/modules/black_market_additions/market_items/weapons.dm

```diff
@@ -0,0 +1,103 @@
+/datum/market_item/weapon
+	category = "Weapons"
+
+/datum/market_item/weapon/bolt_rifle
+	name = "Worn Sportiv Rifle"
+	desc = "A bunch of corrupt imperial law officials have marked a shipment of those as 'defunct' and sold them to us. They still jam like a bitch though."
+	item = /obj/item/gun/ballistic/rifle/boltaction
```

Comment:
if this is supposed to be the bad mosin you should make it the surplus subtype

Reply:
I should do that true.